### ISSUE TITLE
Add sector to network payload

### ISSUE THREAD

**Issue by nightsh** [tenders-exposed/elvis-backend-node#24]
We have a "Sector" field missing for the new design of networks list page. It seems to be a rough generalization of what the network is about, although one might want to double check this with @chutko or @zufanka before adding it.

For the record, the new field was added in [this diff](https://github.com/tenders-exposed/elvis-ember/pull/386/files#diff-ebbe5598ab0481ab912f9edbe9a5094eR26) and it looks like this:
![image](https://user-images.githubusercontent.com/1579997/33488630-f371b8f4-d6b9-11e7-80f6-122667bc1842.png)

**Comment by georgiana-b** [comment]
@zufanka What should the sector be?

**Comment by zufanka** [comment]
sector is the name of the CPV code.
There also should be the CPV code in the view above @chutko

**Comment by georgiana-b** [comment]
Then this is not really an issue since the filter CPVs are already part of the network. 

You will just have to make an extra request  on `/cpvs` for the full names @nightsh .

I'm closing this.

@zufanka 

> There also should be the CPV code in the view above @chutko

☝️ sounds like something important you might want to move to a frontend issue.

**Comment by nightsh** [comment]
I do not agree with this, since it would mean making an extra request / an extra round trip to the server for adding one word / phrase to the visual interface that doesn't seem to counterweight the hassle. I'd say we either drop the "Sector" or we include the name of the CPV code with the most contracts into one of the network's properties. Reopening as up for debate, tagging as "question" as well.

**Comment by chutko** [comment]
what if there are more CPVs selected? 1-3 is fine, but what if there are more? in the component (see the bottom of https://tenders.exposed/examples), the intention was to just display number of CPVs and have a bubble displaying list of the CPVs on mouse over, something like bootstrap popover: http://www.ember-bootstrap.com/#/components/popover
however, i dont know about any component within elvis which would provide this functionality

we can either install something or leave it for future migration to bootstrap which im very serious about
